lib/src/utils.dart

```diff
@@ -1,4 +1,24 @@
 import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+Map<String, String> namedColors = {
+  "White": "#FFFFFF",
+  "Silver": "#C0C0C0",
+  "Gray":	"#808080",
```

Comment:
Not sure if you tried to align the hex codes, but it looks weird now - I would not even bother with this at all.

Reply:
Really odd... There were \t (tab) characters lurking in there but in Android Studio it just showed as a space ?? This alignment weirdness should be fixed now.